app/controllers/api/authentications_controller.rb

```diff
@@ -1,4 +1,11 @@
 module Api
   class AuthenticationsController < BaseController
+    def edit_resource(type, id, data)
+      auth = resource_search(id, type, collection_class(:authentications))
+      task_id = auth.update_in_provider_queue(data)
+      action_result(true, "Updating Authentication with id #{id}", :task_id => task_id)
+    rescue => err
+      raise "Could not update Authentication - #{err}"
```

Comment:
same idea for bulks, if edits for authentications return action result, so are the failures. Thanks.